dts/bindings/arm/arm,dtcm.yaml

```diff
@@ -4,7 +4,7 @@ description: Cortex-M DTCM (Data Tightly Coupled Memory)
 
 compatible: "arm,dtcm"
 
-include: base.yaml
+include: [base.yaml, linker.yaml]
```

Comment:
I know this is used when generating the linker script, but seen from devicetree point of view, I don't think it should be using name linker.

How would `mem-region.yaml` sound ?

Not blocking comment, just a suggestion.

Reply:
makes sense